src/components/Check/Check.scss

```diff
@@ -10,25 +10,37 @@
 $checkBoxHeight: 20px;
 
 .ms-Check {
-  display: inline-block;
-  cursor: default;
-  vertical-align: top;
-  width: $checkBoxHeight;
-  height: $checkBoxHeight;
+  font-family: FabricMDL2Icons;
+  line-height: 1;
   text-align: center;
+  width: 18px;
+  height: 18px;
+  vertical-align: top;
   position: relative;
+  cursor: default;
```

Comment:
A lot of the styles are copied off of onedrive.live.com.  cursor:default seems unnecessary and doesn't have an effect, so I'll remove it in my next commit.